packages/replay/src/integration.ts

```diff
@@ -216,14 +216,18 @@ Sentry.init({ replaysOnErrorSampleRate: ${errorSampleRate} })`,
   }
 
   /**
-   * Immediately send all pending events.
+   * If not in "session" recording mode, flush event buffer which will create a new replay.
```

Comment:
Hmm, not sure I fully understand this comment. If you are in error mode, will this create a new replay? Maybe we can think of a clearer comment here, e.g. "will send an initial recording" or something along these lines? not totally sure...!

Reply:
Yeah if you are in buffer mode, flush will create a new replay.